dist/scroll.js

```diff
@@ -461,14 +461,15 @@
 
 	        var originalTarget = target; // Check if we specified an offset
 
-	        var offset = options['offset'] ? parseInt(options['offset'], 10) : 0;
+	        var offset = options.offset ? parseInt(options.offset, 10) : 0;
```

Comment:
This was auto fixed by lint. It should be fine though, right? Or did you use that syntax for any particular reason?